test/test_jit_llga_fuser.py

```diff
@@ -1,51 +1,106 @@
 # Owner(s): ["module: mkldnn"]
+import sys
 import torch
 import unittest
 import itertools
 
 import torch.nn as nn
 import torch.nn.functional as F
+import torch.fx.experimental.optimization as optimization
 from torch.testing._internal.jit_utils import JitTestCase
 from torch.testing._internal.common_utils import run_tests, TEST_SCIPY, IS_WINDOWS, IS_MACOS
+from torch.testing._internal.common_device_type import (
+    instantiate_device_type_tests,
+    onlyCPU,
+    dtypes
+)
+
+def is_avx512_supported():
+    if sys.platform != 'linux':
+        return False
+    with open("/proc/cpuinfo", encoding="ascii") as f:
+        lines = f.read()
+    return "avx512" in lines
+
+IS_AVX512_UNSUPPORTED = not is_avx512_supported()
 
 LLGA_FUSION_GROUP = 'prim::oneDNNFusionGroup'
 LLGA_NOT_ENABLED = not torch._C.has_mkldnn or IS_WINDOWS or IS_MACOS
 
-
-def warmup_forward(f, *args, profiling_count=2):
+def warmup_forward(f, *args, profiling_count=3):
     for i in range(profiling_count):
         results = f(*args)
 
     return results
 
-
 class JitLlgaTestCase(JitTestCase):
+    original_autocast_mode = False
```

Comment:
Is this line still needed since you've defined `self.original_autocast_mode` later on?

Reply:
Thanks for catching this!